networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc

```diff
@@ -11,7 +11,7 @@ The Kubernetes NMState Operator provides a Kubernetes API for performing state-d
 
 [IMPORTANT]
 ====
-Red Hat supports the Kubernetes NMState Operator in production environments on bare-metal, IBM Power, IBM Z, LinuxONE, and VMware vSphere installations.
+Red Hat supports the Kubernetes NMState Operator in production environments on bare-metal, IBM Power, IBM Z, LinuxONE, VMWare vSphere, and OpenStack installations.
```

Comment:
```suggestion
Red Hat supports the Kubernetes NMState Operator in production environments on bare-metal, IBM Power, IBM Z, LinuxONE, VMware vSphere, and OpenStack installations.
```
According to their website it is written "VMware" (https://www.vmware.com/company.html)

Reply:
iNtErEsTiNg...updating!